### DISCUSSION TITLE
Can I use Snipe-It for assets across multiple departments

### DISCUSSION THREAD

**Discussion by johnroberts2k** [snipe/snipe-it#10485] [Q&A]
I have just installed Snipe-IT. Is it possible to use the app to manage assets across multiple departments? for example we have the I.T department which has hardware, software and consumables. But then we have other departments with their own assets. Ideally I want to be able to manage all assets belonging to the organisation in the same system.

**Comment by yosiasz** [comment]
yes you can! do you want one department to not see another department's assets?

**Reply by johnroberts2k** [reply]
Thanks for your reply. yes ideally want to prevent one department accessing another departments assets. But then also want auditor user accounts that can view all assets across the organisation without being able to make any changes. Is this possible?

**Reply by dfitelw** [reply]
I don't see how roles and such can be used in this way personally, but I wonder if you could "hack" this behavior in using the "Company" functionality?

**Reply by yosiasz** [reply]
Assign auditors to be associated to no company and they will be able to see all companies

**Reply by tpaljor2010** [reply]
@yosiasz , can you please explain how that can be done?

**Reply by yosiasz** [reply]
@tpaljor2010 Sorry I have not be used this wonderful tool for ages now, forgot how

**Reply by a-carrig** [reply]
I can't find anywhere to create auditor accounts.

**Reply by snipe** [reply]
<img width="387" alt="Create_New_Group____Snipe-IT_Demo" src="https://github.com/user-attachments/assets/404bd734-cff3-417b-bced-42923824d0a1" />

**Reply by a-carrig** [reply]
Thanks.  I appreciate the help as I'm learning the program.  I have setup auditors but unfortunately it doesn't solve my need for users to be able to see assets from all companies.
